src/wechaty.ts

```diff
@@ -1,19 +1,18 @@
 /**
- *   Wechaty - https://github.com/chatie/wechaty
  *
  *   @copyright 2016-2017 Huan LI <zixia@zixia.net>
  *
- *   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Why do you change the comment format at here?

Do not include unrelated changes.